src/job-exporter/src/docker_inspect.py

```diff
@@ -75,8 +75,11 @@ def parse_docker_inspect(inspect_output):
             # for kube-launcher tasks
             if k == "FC_TASK_INDEX":
                 m["PAI_TASK_INDEX"] = v
-            elif k == "NVIDIA_VISIBLE_DEVICES" and v != "all" and v != "void":
-                m["GPU_ID"] = v
+            else:
+                if k == "NVIDIA_VISIBLE_DEVICES" and v and v != "all" and v != "void":
+                    m["GPU_ID"] = v
+                if k == "PAI_AMD_VISIBLE_DEVICES" and v:
```

Comment:
those two env vars exist at the same time, change the second `if` to `elif`?

Reply:
or decide by `get_gpu_vendor()`?

Reply:
and what if `NVIDIA_VISIBLE_DEVICES` is `all` or `void`, will skip this env?
pls also add the [`none` case](https://github.com/NVIDIA/nvidia-container-runtime#nvidia_visible_devices).

Reply:
Sure, those two env vars will exist at the same time. But only one will has the value. If all contain the value, it should be mis-config. Will add log for this situation.

In previous logic, if `NVIDIA_VISIBLE_DEVICES` is `all` or `void`, will skip this env. I don't change the logic here. Will add `none` here.

Reply:
For AMD GPU, seems both `NVIDIA_VISIBLE_DEVICES` and `PAI_AMD_VISIBLE_DEVICES` will be set....
Will correct this.